static/docs/user-guide/install/completion.md

```diff
@@ -0,0 +1,154 @@
+# Tab Completion
+
+Enjoy working with DVC faster and with fewer typos!
```

Comment:
Let's preserve the original name, ** Shell Autocomplete**. Or perhaps "Shell Auto Completion" for an improvement. These seem to be the most used terms out there.

Also, should remove the original doc, or they will be duplicated. Still here: https://dvc-org-pr-695.herokuapp.com/doc/user-guide/autocomplete

Reply:
> Let's preserve the original name, ** Shell Autocomplete**. Or perhaps "Shell Auto Completion" for an improvement.

Or perhaps "Shell Tab Completion".

> should remove the original doc, or they will be duplicated

Let's postpone this for another PR that follows this one.

Reply:
Or **Command-line completion**. Nah let's just get it done 🙂 @shcheklein which term do you vote for?

Reply:
I would vote for something like https://medium.com/@dilankam/enable-auto-completion-of-git-commands-on-mac-os-mojave-10-14-2d692c100bc - `Enable Auto Completion` or ``Enable Autocomplete` (I'm not sure which one is a better form for a native speaker) - I think most users should understand this. We can also, mention on the index page and explain it using other words (like Tab Completion). @gagejustins what would be you take on this, I wonder? :)

Reply:
I guess the currently use terms **Shell Completion** nav link and title **Shell Tab Completion* work fine.

Reply:
Reopening due to Ivan's last comment, which I just noticed. Also curious about @gagejustins' input here.

Reply:
Hey all, sorry for the delay - was at our company offsite. What we're talking about here is traditionally referred to as some variation of "auto complete" - using the "tab" key is often the *way* that you do it in an editor, but it's not part of the name